src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -129,11 +133,31 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     for {
       _ <- assertWorkflowExistence(workflowId)
       _ <- assertCallExistence(workflowId, callFqn)
-      (wf, call) <- assertCallFqnWellFormed(callFqn)
+      (wf, call) <- Future.fromTry(assertCallFqnWellFormed(callFqn))
       callStandardOutput <- Future.successful(backend.stdoutStderr(workflowId, wf, call))
     } yield callStandardOutput
   }
 
+  private def workflowStdoutStderr(workflowId: WorkflowId): Future[Map[FullyQualifiedName, StdoutStderr]] = {
+    def logMapFromStatusMap(statusMap: Map[FullyQualifiedName, ExecutionStatus]): Try[Map[LocallyQualifiedName, StdoutStderr]] = {
+      Try {
+        val callsToPaths = for {
+          (call, status) <- statusMap.toSeq
+          if status == ExecutionStatus.Done
```

Comment:
It should include any statuses for which we have stdout/stderr.  If `Failed` is one of those statuses, then yes.